transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/shyrka/WebMessagingJson.kt

```diff
@@ -17,14 +17,13 @@ import com.genesys.cloud.messenger.transport.shyrka.receive.UploadFailureEvent
 import com.genesys.cloud.messenger.transport.shyrka.receive.UploadSuccessEvent
 import com.genesys.cloud.messenger.transport.shyrka.receive.WebMessagingMessage
 import kotlinx.serialization.SerializationException
-import kotlinx.serialization.decodeFromString
 import kotlinx.serialization.json.Json
 
 internal object WebMessagingJson {
 
     val json = Json {
         ignoreUnknownKeys = true
-        useAlternativeNames = false
+        classDiscriminator = "messageClass"
```

Comment:
I am a little concern about this change. It will effect all our parsing objects. Can you extend why we need to remove the useAlternativeNames?